### DISCUSSION TITLE
Anyone notice their padding in android is 2x what the apple padding is?

### DISCUSSION THREAD

**Discussion by dutchkillscreative** [jaredh159/tailwind-react-native-classnames#165] [Q&A]
Does this have to do with retina? I don't understand why every padding component I need to write seems to require `ios:p-4 android:p-2` to achieve the same visual result. Brand new React Native template.

Edit: Testing on a simulated 13 Pro Max and Nexus 5. But I'm seeing the same results when usaing an actual Moto G4 Plus (old I know) and a iPhone 13 mini

**Comment by jaredh159** [comment]
I do not think it has to do with retina DPI things.  I'm not sure exactly why you're seeing this, but I've definitely noticed that there are a whole host of subtle rendering differences between the platforms that you either have to accept or work around.  I don't think it's an issue with this library per se.  You can test that by eliminating the call to `tw` for a component, and just pass raw RN style objects, and you should see the same issue.
